kvm-install-vm

```diff
@@ -345,6 +346,14 @@ function fetch_images ()
             DISK_FORMAT=qcow2
             LOGIN_USER=debian
             ;;
+        debian10)
+            QCOW=debian-10-openstack-amd64.qcow2
+            OS_TYPE="linux"
+            OS_VARIANT="debian10"
```

Comment:
Note: "auto" does not work here. The debian guest requires the os variant in order to setup the kernel options, otherwise the network interface will not be started!

Reply:
Thanks for pointing this out. I had trouble with a previous version of Debian and I didn't figure out why. Probably could have been related to this, or something similar.